src/PrestaShopBundle/Resources/config/routing/admin/improve/international/translations.yml

```diff
@@ -34,17 +35,12 @@ admin_international_translations_add_update_language:
     defaults:
         _controller: PrestaShopBundle:Admin/Translations:addUpdateLanguage
         _legacy_controller: AdminTranslations
+        _legacy_link: AdminTranslations:submitAddLanguage
 
 admin_international_translations_copy_language:
     path: /copy
     methods: [POST]
     defaults:
         _controller: PrestaShopBundle:Admin/Translations:copyLanguage
         _legacy_controller: AdminTranslations
-
-admin_international_translations_export_language:
```

Comment:
this route was not used anywhere, and the controller action does not exist as well. there is another action for exporting language, so I think this remained here after rebasing.

Reply:
Indeed it has been removed in `develop` by https://github.com/PrestaShop/PrestaShop/pull/13200